### PR TITLE
Replace `== bool` and `~= bool`

### PR DESCRIPTION
**Pull request by fryguy503**
Cleanup of bool comparisons.

### CODE REVIEW

File: airplane/The_Spiroc_Lord.lua [hunk 1]
```diff
@@ -1,7 +1,7 @@
 function event_death_complete(e)
 	local qglobals = eq.get_qglobals();
 	
-	if(qglobals["sirran"] ~= "5" and eq.get_entity_list():IsMobSpawnedByNpcTypeID(71009) == false) then
+	if qglobals["sirran"] ~= "5" and eq.get_entity_list():IsMobSpawnedByNpcTypeID(71009) then
```
Comment (Kinglykrab) [inline airplane/The_Spiroc_Lord.lua:4]: This checked for false before, we’re not checking for false now.

Comment (fryguy503) [reply airplane/The_Spiroc_Lord.lua:4]: Thanks - Resolved

**Review by joligario — APPROVED** [review]
Lots of work - nice! I didn't see any issues.

File: bloodfields/#Elite_Dragorn_Jekisia.lua [hunk 2]
```diff
@@ -17,18 +17,18 @@ after 20 minutes at a location she will wipe her agro and reset to bind
 
 
 
-local Agro=0;
+local Agro = false;
```
[the remark this answers is not in the document]
Comment (fryguy503) [reply bloodfields/#Elite_Dragorn_Jekisia.lua]: done

**Pull request merged**